### PR TITLE
Add seats.io types

### PR DESCRIPTION
**Pull request by mortendevold**
Move existing types to new repository.

### CODE REVIEW

**Comment by mortendevold** [comment]
[Make seatsio-types an open-source project](https://trello.com/c/z2RAaWgd/6015-make-seatsio-types-an-open-source-project)

File: .github/workflows/publishTypes.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          registry-url: https://registry.npmjs.org/
+
+      - name: Build types
+        working-directory: ./ui/seatsio-types
+        run: |
+          yarn install
+          yarn build
+        
+      - name: Checkout latest master
+        working-directory: ./ui/seatsio-types
```
Comment (mroloux) [inline .github/workflows/publishTypes.yml:40]: We'll need to fix those references to the ui folder

**Review by mroloux — CHANGES_REQUESTED** [review]
This is a good start 👍 

I would add:
- a license file
- a GitHub Actions flow that builds the code when opening a PR, and when merging to master

File: .github/workflows/publishTypes.yml (hunk 1 above)
Comment (mortendevold) [reply .github/workflows/publishTypes.yml]: Changed to work on root folder. License file is already present on master (only file there, MIT license).

**Review by mroloux — CHANGES_REQUESTED** [review]
`publishTypes` runs when you manually start it, after the PR has been merged. Let's also add a separate Github Actions flow, which builds the types when opening a PR, and when merging to master. That should help us spot issues soon.

**Comment by mortendevold** [comment]
Absolutely, I've added a workflow that will run test and build, as well as disabling JS emit when compiling for tests only.

**Review by mroloux — APPROVED** [review]

**Pull request merged**